apps/src/weblab/WebLab.js

```diff
@@ -98,6 +98,7 @@ WebLab.prototype.init = function (config) {
   config.centerEmbedded = false;
   config.wireframeShare = true;
   config.noHowItWorks = true;
+  config.baseShareUrl = 'https://codeprojects.org';
```

Comment:
Correct, this will not. I verified that we do not have staging or test load balancers set up for these other environments, because we cloned the setup of hourofcode.com. If/when we do that, I figured that I would add a variable as you describe.